ngshare/ngshare.py

```diff
@@ -86,7 +96,10 @@ def json_files_pack(self, file_list, list_only):
                 'checksum': i.checksum,
             }
             if not list_only:
-                entry['content'] = base64.encodebytes(i.contents).decode()
+                storage_path = self.application.storage_path
+                actual_path = os.path.join(storage_path, i.actual_name)
+                content = open(actual_path, 'rb').read()
```

Comment:
might want to close the file afterwards, or just use `with open(...) as f:`

Reply:
I think `open(...)` is not saved in Python so is auto-closed.